velox/row/UnsafeRowFast.cpp

```diff
@@ -59,24 +59,41 @@ void UnsafeRowFast::initialize(const TypePtr& type) {
     case TypeKind::ARRAY: {
       auto arrayBase = base->as<ArrayVector>();
       children_.push_back(UnsafeRowFast(arrayBase->elements()));
-      childIsFixedWidth_.push_back(
-          arrayBase->elements()->type()->isFixedWidth());
+      if (arrayBase->elements()->type()->kind() == TypeKind::HUGEINT) {
```

Comment:
This logic repeats. Let's extract a helper function to improve readability: bool isFixedWidth(const TypePtr& type)